docs/developer-docs/smart-contracts/compile.mdx

```diff
@@ -0,0 +1,75 @@
+---
+keywords: [beginner, tutorial, compile]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
+
+# Compile
+
+<MarkdownChipRow labels={["Beginner", "Tutorial"]} />
+
+## Overview
+
+After you have written the code for your project's <GlossaryTooltip>canisters</GlossaryTooltip>, you need to compile the code into a WebAssembly module before it can be deployed on ICP. 
+
+## How code is compiled to Wasm
+
+WebAssembly (Wasm) is platform independent and designed to run on a stack-based virtual machine. Wasm itself is not a programming language; it is the result of a programming language being comnpiled and converted into executable machine code. When canister code is compiled into Wasm, the system compiler creates a Wasm module in either a binary format (.wasm) or a compressed binary (.wasm.gz).
```

Comment:
```suggestion
WebAssembly (Wasm) is platform independent and designed to run on a stack-based virtual machine. Wasm itself is not a programming language; it is the result of a programming language being compiled and converted into executable machine code. When canister code is compiled into Wasm, the system compiler creates a Wasm module in either a binary format (.wasm) or a compressed binary (.wasm.gz).
```

Reply:
Removed this content in favor of revised content provided by Ulan